Documentation/ldap-connector.md

```diff
@@ -0,0 +1,86 @@
+# Overview of the `ldap` Connector
```

Comment:
Title should be something like "Authentication through LDAP"

Then we would have a `## Overview` subsection